docs/porting/target/bootloader.md

```diff
@@ -8,10 +8,8 @@ The target needs to support the Flash HAL API. You can find the guide to porting
 
 ## Porting the bootloader
 
-You can learn to port the Update Client bootloader to new targets in [the porting guide](https://cloud.mbed.com/docs/latest/porting/porting-the-device-management-update-client.html).
+You can learn to port the Update client bootloader to new targets in [the porting guide](https://www.pelion.com/docs/device-management/current/porting/porting-the-device-management-update-client.html).
 
-An example walkthrough of porting the bootloader to the K64F platform can be found [here](https://cloud.mbed.com/docs/latest/porting/update-k64f-port.html).
-
-## Creating and using a new bootloader
+An example walkthrough of porting the bootloader to the NUCLEO-F411RE target can be found [here](https://www.pelion.com/docs/device-management/current/porting/porting-a-new-target-for-mbed-os-using-sotp.html).
```

Comment:
Let's avoid naming specific targets if possible. What if we say "For details, see our guide about [porting the bootloader to a target](https://www.pelion.com/docs/device-management/current/porting/porting-a-new-target-for-mbed-os-using-sotp.html)."?

Reply:
Yeah, and that way we link on a more meaningful phrase.